source/physics/UniformElectricDriftField.cc

```diff
@@ -84,23 +87,18 @@ namespace nexus {
     // Set the new time and position of the drifting charge
     xyzt.set(time, position);
 
+    G4double rnd = -lifetime_ * log(G4UniformRand());
```

Comment:
Do we need a catch here in case of log(0)?

Reply:
It seems that `0` is excluded, see, for instance, https://geant4-forum.web.cern.ch/t/what-is-the-range-of-numbers-generated-by-g4uniformrand/5187/2.